zeek/Data-Type-Compatibility.md

```diff
@@ -17,9 +17,10 @@ As the Zed data model was in many ways inspired by the
 [Zeek TSV log format](https://docs.zeek.org/en/master/log-formats.html#zeek-tsv-format-logs),
 the rich Zed storage formats ([ZSON](../docs/formats/zson.md),
 [ZNG](../docs/formats/zng.md), etc.) maintain comprehensive interoperability
-with Zeek. However, when Zeek is configured to output its logs in JSON or
-NDJSON formats, much of the rich type information is lost in translation. On
-the other hand, Zeek TSV can be converted to Zed storage formats and back to
+with Zeek. When Zeek is configured to output its logs in JSON or
+NDJSON formats, much of the rich type information is lost in translation, but
```

Comment:
Zeek can emit NDJSON but not JSON.
```suggestion
with Zeek. When Zeek is configured to output its logs in
NDJSON format, much of the rich type information is lost in translation, but
```

Reply:
@nwt: Yeah, fair point. My use of "JSON" was a hold-over from an attempt to write in the wording that Zeek's docs use to describe itself (i.e. the option in their ASCII logger is called `use_json`, not `use_ndjson`), and at one time when I wrote docs I sometimes chose presumed audience familiarity over technical purity. However, given the direction things are headed, your proposed purity sounds better.